### PR TITLE
Adding CONTRIBUTING file.

### LINKED ISSUE #10: Creating a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by GalOved**
Adding CONTRIBUTING file to the project.

Close #10

### CODE REVIEW

**Review by ohadrevach — APPROVED** [review]
Looks fine to me

**Review by GilShechter — DISMISSED** [review]

File: CONTRIBUTING.md
Comment (natifridman) [inline CONTRIBUTING.md:39]: These keywords should be used in the PR to link an issue, and not in the PR itself

File: CONTRIBUTING.md
Comment (natifridman) [inline CONTRIBUTING.md:74]: I would add that the CI tests passed and didn't fail

**Review by natifridman — COMMENTED** [review]
Nice work, added some comments.

I would also add something about the number of commits in the PR.

**Review by ohadrevach — APPROVED** [review]

**Comment by natifridman** [comment]
Please try not to force push during the review process. The reviewers can't see what you changed from the last time.
You can squash the commits after the PR is approved before the merge

**Review by efraimvis — APPROVED** [review]

**Review by WaseemAbuSobe — DISMISSED** [review]

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:19]: The hyperlinks seem broken here, from looking at the "rich diff"

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:59]: missing space here

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:56]: please fix newline spacing, looking at the rich diff

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:67]: please fix newline spacing, looking at the rich diff

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:103]: Those are rules of what NOT to follow.

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:1]: missing a space here

**Review by efraimvis — APPROVED** [review]

File: CONTRIBUTING.md
Comment (natifridman) [inline CONTRIBUTING.md:114]: Maybe worth to add somewhere in this doc who are the maintainers

**Comment by GalOved** [comment]
@natifridman I updated the CONTRIBUTING file according to the reviewes

File: CONTRIBUTING.md
Comment (liornoy) [reply CONTRIBUTING.md:67]: @GalOved I think you missed this section :)
Please fix the spacing between the lines.

**Comment by liornoy** [comment]
@GalOved I think you missed the "Pull Requests Standards" section :)
Please fix the spacing between the lines.

**Comment by GalOved** [comment]
@liornoy l fix the "Pull Requests Standards" section.

**Comment by liornoy** [comment]
Please squash the commits so we'll remain with only one.
(As all of the current 4 commits are changing only 1 file)

**Comment by GalOved** [comment]
@liornoy i squash all the commits together.

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:140]: Without an existing e-mail, this section doesn't make sense.

**Comment by GalOved** [comment]
@liornoy hi lior I fix the help section, delete that section becase it is not relvent for now.

File: CONTRIBUTING.md
Comment (liornoy) [inline CONTRIBUTING.md:19]: This quick link is not relevant anymore.

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:29]: Maybe mention which version is most recommended to be used (hint, the one all of you are using)

Comment (svatares) [reply CONTRIBUTING.md:29]: Also, even better to add direct links to the downloads so it will be easier for a new contributor to start

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:31]: This is vague, from which location does the vagrant up should run? (It is a basic thing that everyone should know, but if you weren't familiar with Vagrant, what would you do?)

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:39]: This sentences are repeating themselves. Please rephrase it to make it more simplified a readable

Comment (svatares) [reply CONTRIBUTING.md:39]: Also, you're mentioning here the acronym `PR` where you only introduce it later on in the next section...

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:48]: Is this mandatory for any PR you're about to create?

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:49]: Fork `the` repository, not `a`

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:51]: ... `our` local machine -> ... `your` local machine 

And overall you're describing several actions on one bullet point, where it should probably only mention one action per bullet

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:62]: ... pull request `is`: -> ... pull request `are`:

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:64]: Can maybe rephrased to be more accurate and show a positive idea (instead of negative idea - `bad language`)

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:65]: ... `be a` meaningful ... -> ... `have a` ...

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:94]: `Our standards` - does it really needed here?

File: CONTRIBUTING.md
Comment (svatares) [inline CONTRIBUTING.md:103]: `Our standards` - does it really needed here?

File: CONTRIBUTING.md
Comment (svatares) [reply CONTRIBUTING.md:114]: I can see that it's mentioned in the `Maintainers` section below

**Review by svatares — COMMENTED** [review]
Nice work, left a few comments

File: CONTRIBUTING.md
Comment (GalOved) [reply CONTRIBUTING.md]: Hi, @svatares I corrected the change according to your suggestion and also rewrote part of the file.
You can see details about the things that were written in commit.

**Review by GilShechter — APPROVED** [review]
Looks great, nice job

**Comment by liornoy** [comment]
I think the CONTRIBUTING file looks good.

**Comment by liornoy** [comment]
@ohadrevach @WaseemAbuSobe Please go through the CONTRIBUTING file and review it,
and leave a comment if you agree/disagree with the content.
We must have everyone's consent before merging.

**Comment by svatares** [comment]
Nice work on changes, I resolved all of my comments, and most of the comments in general. @natifridman there are a couple of comments you left that were addressed, please see if those were resolved.
And in order to finish reviewing this PR we're missing @ohadrevach 's review

**Review by svatares — APPROVED** [review]

**Review by ohadrevach — APPROVED** [review]
looks good to me

**Review by natifridman — APPROVED** [review]
Approved

**Pull request merged**
